pkg/image/api/v1/trigger/trigger.go

```diff
@@ -0,0 +1,24 @@
+package trigger
```

Comment:
These triggers are image specific.  The struct is under the API because they are serialized into annotations for the generic path.